packages/commons/src/auth/jwk.ts

```diff
@@ -64,3 +66,17 @@ export function sortJWK(jwk: JsonWebKey): JsonWebKey {
       {}
     );
 }
+
+export function getJwksClient(): JwksClient[] {
+  const config = JWTConfig.parse(process.env);
```

Comment:
I would pass config as a parameter so that we align the auth middleware to the standard we follow in the project: we only `parse(process.env)` in `config/config.ts` files outside any function, so that parsing can fail only at startup - I know there already was a `.parse` in the middleware but if you agree I would refactor to accept accept a `config` parameter in the middleware function

Reply:
With this commit [bab7433](https://github.com/pagopa/interop-be-monorepo/pull/1002/commits/bab74331b5a944162078a9525e908797f99e119f) configuration become a function parameter.